Intersect.Client/Core/Fade.cs

```diff
@@ -21,28 +23,38 @@ public enum FadeType
 
         private static float sFadeAmt;
 
-        private static float sFadeRate = 3000f;
+        private static float sFadeRate = ClientConfiguration.DEFAULT_FADE_SPEED_MS;
```

Comment:
If we now always have to pass this in, what's the point of setting it to an initial value here?

Reply:
Also rename to duration

Reply:
done, good catch